wiki/Tournaments/Mappool/en.md

```diff
@@ -0,0 +1,95 @@
+---
+stub: true
+tags:
+  - map pool
+  - modpool
+  - mod pool
+  - mappool slot
+  - map pool slot
+  - pool slot
+  - rice
+  - LN
+  - hybrid
+  - SV
+  - no mod
+  - NM
+  - free mod
+  - force mod
+  - forced mod
+  - freemod
+  - forcemod
+  - forcedmod
+  - FM
+  - mixed mod
+  - MM
+  - tiebreaker
+---
+
+# Mappool
+
+A **mappool** is a collection of [beatmaps](/wiki/Beatmap) used in [tournament](/wiki/Tournaments) matches. These mappols may be divided into sections based on which [mods](/wiki/Gameplay/Game_modifier) are used, in which case they are sometimes called *modpools*, or by type of beatmap. The individual beatmaps are said to occupy a numbered *mappool slot* within each section of the mappool. People who create mappools are called *mappoolers* or *mappool selectors*, and the process itself is called *mappooling*.
+
+## Mappool sections
+
+Tournaments except osu!mania ones typically include *Hidden*, *Hard Rock*, and *Double Time* sections of the mappool, which as their names suggest require the respective mods to be active. osu!mania tournaments instead use *Rice*, *LN*, *Hybrid*, and *SV*.
+
+### No Mod
+
+**No Mod** (***NM***) refers to not using any mods. Many tournaments require some mods to be used by default as part of their rules or format, such as [No Fail](/wiki/Game_modifier/No_Fail) or [ScoreV2](/wiki/Game_modifier/ScoreV2), which become exceptions to this notion.
+
+### Free Mod
+
+**Free Mod** (***FM***) refers to being free to choose any mod or mod combination. Some tournaments also provide rules that specify additional criteria such as what mods are allowed and in what combinations, or whether having no mods is allowed when Free Mod is specified.
+
+### Force Mod
+
+**Force Mod** (***FM***) is otherwise the same as [Free Mod](#free-mod), except that it specifies that using at least one mod is mandatory.
+
+### Mixed Mod
+
+**Mixed Mod** (***MM***) means each player in the team has to pick one unique mod or mod combination each. The specific requirements may vary between tournaments.
+
+### Rice
+
+**Rice** is an osu!mania community term for regular notes, in reference to regular notes looking like rice. Rice maps thus contain a lot of regular notes.
+
+### LN
+
+**LN** in osu!mania stands for long note, and thus LN maps contain a lot of them.
+
+### Hybrid
+
+**Hybrid** beatmaps contain both regular and long notes, i.e. is a category in between Rice and LN.
+
+### SV
+
+**SV** stands for slider velocity, which in osu!mania affects scroll speed. SV beatmaps make use of scroll speed variation gimmicks.
```

Comment:
these would be all good entries for glossary (if we didn't tear it down and agree on its non-existence)...
however! we don't have articles on o!m hit objects, so these all seem like good topics to explore.

come to think of it, if your task is to describe which map pool categories are there at all, I'd try dividing them into three categories (three article sections, even)
- requiring a single specific mod
- requiring special mod selection rules (FM/NM/MM/w/e)
- containing specific map types -- here goes all o!m stuff and maybe tiebreakers (however that would be a stretch)